webpack/assets/javascripts/react_app/components/AuditsList/ShowOrgsLocs.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Col } from 'patternfly-react';
+import ShowTaxonomyInline from './ShowTaxonomyInline';
+
+const ShowOrgsLocs = ({
+  isOrgEnabled = false,
```

Comment:
Please add Proptype validation for the props. plus with proptype you can add a default value for a prop:

```javascript
 ShowOrgsLocs.defaultProps = {
  isLocEnabled: false
};
```